setup.py

```diff
@@ -76,8 +76,14 @@
                 # Debian's suitesparse-dev installs to
                 # /usr/include/suitesparse
                 "/usr/include/suitesparse",
+                # macOS's suitesparse installs to
+                # /usr/local/include/suitesparse
+                "/usr/local/include/suitesparse",
+            ],
+            library_dirs=[
+                # macOS's suitesparse installs to /usr/local/lib
+                "/usr/local/lib",
```

Comment:
This doesn't add anything thing since /usr/local/ is already in the standard search path. The problem mentioned in #67 was that they has used macports (honestly didn't know that was still a thing) which installs to /opt which isn't in the standard search path.

I think the best option to make this all compatible for the end user is to allow specification of these at runtime through environment variables. Something like


```python
INCLUDE_DIRS = [
    np.get_include(),
    sys.prefix + "/include",
    # Debian's suitesparse-dev installs to
    # /usr/include/suitesparse
    "/usr/include/suitesparse",
]

LIBRARY_DIRS = []

user_include_dir = os.getenv("SUITESPARSE_INCLUDE_DIR")
user_library_dir = os.getenv("SUITESPARSE_LIBRARY_DIR")
if user_include_dir:
    INCLUDE_DIRS.append(user_include_dir)

if user_library_dir:
    LIBRARY_DIRS.append(user_library_dir)

```
Then later in `setup`
```python
...
include_dirs = INCLUDE_DIRS,
library_dirs = LIBRARY_DIRS,
...

Reply:
Actually I have just implemented this here #71

Reply:
I think this is a versatile solution.